### DISCUSSION TITLE
security and auth collection creation with email and password

### DISCUSSION THREAD

**Discussion by airnh** [pocketbase/pocketbase#2037] [Q&A] [answered]
One behavior I noticed with the social login, it automatically creates a user if the user doesn't exist. To be able to do this, the API rule for "create" seems to be public as the new user won't have any identity beforehand.

If so, anyone can send multiple requests to create a new auth record using the endpoint URL with a bunch of different data at a good enough slow rate to bypass auto-cancel. Actually, the rate limit can be murky as the actual users can behave like that too.

So I wondered if the PB can filter out the request domains or IP address?

Please advise.

**Comment by ganigeorgiev** [comment, marked as answer]
Please check https://github.com/pocketbase/pocketbase/discussions/2030
